packages/core/src/rum/instrumentation/resourceTracking/requestProxy/XHRProxy/DatadogRumResource/internalDevResourceBlocklist.ts

```diff
@@ -16,7 +16,7 @@ import type { RUMResource } from '../../interfaces/RumResource';
  * An example URL is http://192.168.1.20:8081/logs or http://10.46.29.155:19000/logs
  */
 const EXPO_DEV_LOGS_REGEX = new RegExp(
-    '^http://((10|172|192).[0-9]+.[0-9]+.[0-9]+|localhost):[0-9]+/logs$'
+    '^http://((10|172|192).[0-9]+.[0-9]+.[0-9]+|localhost|127.0.0.1):[0-9]+/logs$'
```

Comment:
I'm wondering if we have a possibility of having customer server deployment locally as well, this way they may be puzzled why then tracing is not working when they target local instance?